distribution/packages/build.gradle

```diff
@@ -454,7 +454,7 @@ subprojects {
 
     tasks.register("checkNotice") {
       dependsOn buildDist, "checkExtraction"
-      onlyIf {
+      onlyIf("deb project and dpkg exists or rpm project and rpm exists") {
```

Comment:
We can make this message dynamic I think. Something like
```
onlyIf("${project.name.contains('deb') ? 'dpkg' : 'rpm'} exists")
```

Reply:
fixed